backend/src/email/templates/verify-email.pug

```diff
@@ -0,0 +1,17 @@
+extends _layout
+
+block variables
+  - const subject = 'Verify your QuoVote account'
+
+block content
+  .card
```

Comment:
We need a variable called `greeting`. 

In case the variable name is provided: `Hello #{name},` 
Otherwise, we should have a fallback like `Hello!`

Reply:
👌